File: src/Core/Tests/RazorEngine.Core.VB.Tests/TestTypes/Activation/ITextFormatter.vb
Namespace TestTypes.Activation
    ''' <summary>
    ''' Defines the required contract for implementing a text formatter.
    ''' </summary>
    Public Interface ITextFormatter
#Region "Methods"
        ''' <summary>
        ''' Formats the specified value.
        ''' </summary>
        ''' <param name="value">The value to format.</param>
        ''' <returns>The formatted value.</returns>
        Function Format(value As String) As String
#End Region
    End Interface
End Namespace

File: src/test/Test.RazorEngine.Core/TestTypes/Activation/ITextFormatter.cs
namespace RazorEngine.Tests.TestTypes.Activation
{
    /// <summary>
    /// Defines the required contract for implementing a text formatter.
    /// </summary>
    public interface ITextFormatter
    {
        #region Methods
        /// <summary>
        /// Formats the specified value.
        /// </summary>
        /// <param name="value">The value to format.</param>
        /// <returns>The formatted value.</returns>
        string Format(string value);
        #endregion
    }
}
